.azure-pipelines/templates/secure-files.yml

```diff
@@ -1,9 +1,15 @@
+parameters: 
+- name: stageID
+  type: string
+  default: ''
+
 steps:
-  - task: DownloadSecureFile@1
-    inputs: 
-      secureFile: 'local.properties'
-      retryCount: '5'
+- task: DownloadSecureFile@1
```

Comment:
I think there's a problem with the indent here

Reply:
I believe the indents are correct here. The first task gets called regardless of which stage calls it, so it falls directly under the 'steps' line.